operator/pkg/util/kyma_util.go

```diff
@@ -47,7 +47,7 @@ func GetUnstructuredComponentFromTemplate(templates release.TemplateLookupResult
 	}
 
 	desiredComponentStruct := &lookupResult.Template.Spec.Data
-	desiredComponentStruct.SetName(componentName + "-name")
+	desiredComponentStruct.SetName(componentName + kyma.Name)
```

Comment:
To Reviewer: this change avoid unique component name when deploy multiple kyma objects, and due to current load test setup, we relay on `-` to split the component name to adapt into component<->target cluster mapping, so here I haven't use `-` as a separator (just want to avoid maintenance difference between branches).